gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/GradleCacheExplorer.java

```diff
@@ -57,17 +59,36 @@ public final Set<String> getCompletions(Set<String> repoUrls, DependencyGroup in
                 .flatMap(Optional::stream)
                 .collect(Collectors.toSet());
 
+        Set<String> resultsWithStarsIncluded = results.stream()
+                .map(result -> {
+                    int colonIndex = result.indexOf(':');
+                    if (colonIndex != -1) {
+                        return Arrays.asList(result, result.substring(0, colonIndex) + ":*");
+                    }
+                    return Collections.singletonList(result);
+                })
+                .flatMap(Collection::stream)
+                .collect(Collectors.toSet());
+
         if (parsedInput.isEmpty()) {
             return results;
         }
```

Comment:
if this doesn't depend on `resultsWithStarsIncluded` you should move it above it